### ISSUE TITLE
the shape is the same with up to 15 percent difference

### ISSUE THREAD

**Issue by katayooneshkofti** [lululxvi/deepxde#301]
@lululxvi 
firstly, I should thank you for all the responses; I have a question about my problem. I solved a problem with two inputs and also two outputs by Deepxde. The results were pretty satisfactory, but something odd happens in different examples: the output related to the temperature completely fits with the results previously obtained by numerical methods; however, when it comes to the displacement (second output), the shape is totally the same with the numerical results (it follows the same pattern) but the values have about 10-15 percent difference with the numerical results. I have read the FaQ about training loss or other related issues. what do you recommend? should I use more points in the domain, boundary and initial conditions? or should I tune other hyperparameters?

**Comment by lululxvi** [comment]
It is strange that the temperature is accurate, but displacement is not. It is possible that the loss of displacement is not well trained? You can check if each loss term is small.

**Comment by katayooneshkofti** [comment]
@lululxvi 
The more strange part is that the order of loss terms for temperature and displacement is the same and equal to O(10^-6)
I am really confused what approach may help to address this issue?

**Comment by lululxvi** [comment]
No idea what happened. How about trying another but similar and simpler problem first?

**Comment by katayooneshkofti** [comment]
@lululxvi 
 I checked it for simpler problems, the displacement results are satisfactory with up to 5 percent error. Can Assigning more loss weight to the displacement be an option? Does it make sense? 🤔
I have no idea what might help.

**Comment by lululxvi** [comment]
You can definitely try increasing the weight. Or there might be some tiny thing being ignored.

**Comment by katayooneshkofti** [comment]
@lululxvi 
Thank you
